samples/DotNet/Rbac/EventHubsSenderReceiverRbac/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Configuration;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.ServiceBus;
+using Microsoft.ServiceBus.Messaging;
+
+namespace EventHubsSenderReceiverRbac
+{
+    class Program
+    {
+        static void Main()
+        {
+            string tenantId = ConfigurationManager.AppSettings["tenantId"];
+            string clientId = ConfigurationManager.AppSettings["clientId"];
+            string eventHubNamespace = ConfigurationManager.AppSettings["eventHubNamespaceFQDN"];
+            string eventHubName = ConfigurationManager.AppSettings["eventHubName"];
+
+            MessagingFactorySettings messageFactorySettings = new MessagingFactorySettings
+            {
+                TokenProvider = TokenProvider.CreateAadTokenProvider(
+                    new AuthenticationContext($"https://login.windows.net/{tenantId}"),
+                    clientId,
+                    new Uri("http://eventhubs.microsoft.com"),
```

Comment:
JFYI - This is the redirect uri set on the app and doesn't have to be http://eventhubs.microsoft.com. I just happened to put it but you can have any valid uri. 

![image](https://user-images.githubusercontent.com/27301049/33103629-3411a958-ced8-11e7-926e-de5ad12b5c7e.png)